metadata-ingestion/src/datahub/sdk/lineage_client.py

```diff
@@ -144,12 +152,17 @@ def add_dataset_copy_lineage(
                 downstream=downstream,
                 cll_mapping=column_lineage,
             )
+        else:
```

Comment:
unrelated to this feature but added typing check to prevent error (as-is -- it throws `cll` is not defined when putting random type

Reply:
you can use `assert_never` for this